### PR TITLE
Fix/patterns

### PR DESCRIPTION
**Pull request by RWOverdijk**

### CODE REVIEW

**Comment by RWOverdijk** [comment]
I didn't test this yet... I know I should, and I will, but I have a daytime job to attend to now. *sigh*.

File: commands/patterns.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const constants = {
```
Comment (GantMan) [inline commands/patterns.js]: heads up, I don't think this can be in the commands folder, it will be established as a command.  Will have to test, or maybe @skellock knows off the top of his head.

My gut says it will need to move to `../patterns.js`

Comment (skellock) [inline commands/patterns.js]: Because it exports an object I think it'll be ok.  Commands are functions.  Tucking it in `../patterns.js` will be safe.  As will putting under a new sub-directory under commands.  But ya, the *.js located in commands will be attempted to be loaded by gluegun as a potential command.

TL;DR:  Make sure you test it.  I believe it will be ok where it is.

Comment (RWOverdijk) [inline commands/patterns.js]: I'll just move it up one directory to be sure.

Comment (RWOverdijk) [inline commands/patterns.js]: Just tested it, and it works. I'll move it up for pretty reasons

**Comment by RWOverdijk** [comment]
Updated and tested
